docs/book/src/plugins/go-v3-plugin.md

```diff
@@ -0,0 +1,21 @@
+# go/v3
+
+This is the default plugin that is used by Kubebuilder to create operators. It basically scaffolds all the boilerplate code required to create and design operators. Check the following [example](https://github.com/kubernetes-sigs/kubebuilder/tree/master/testdata/project-v3) to view the files created or updated by this plugin when used with the supported subcommands. You can also follow the [Building CronJob](cronjob-tutorial.md) tutorial to get a step-by-step explanation of how the plugin works with different subcommands.
```

Comment:
See:

All Operators are made of [controllers](https://github.com/kubernetes-sigs/controller-runtime), but not all sets of controllers are operators. Operators are sets of controllers that encode human operational knowledge for a specific application (e.g. the Prometheus Operator). Controllers are more general -- for instance, the namespace service account controller isn't an operator because it doesn't have knowledge of a specific application -- it's very general. Kubebuilder helps constructs sets of controllers. Some of those are Operators, some might not be.

So, could we update it accordingly? WDYT?

Reply:
Got it. I will make the changes accordingly